.github/workflows/unit-tests.yaml

```diff
@@ -30,60 +30,6 @@ jobs:
         # Prevent fatal error "detected dubious ownership in repository" from recent git.
         git config --global --add safe.directory "$(pwd)"
 
-    - name: Cache Go dependencies
```

Comment:
I actually don't think we can change anything about the unit tests because they run different things. One runs postgres integration tests and one runs go tests iirc

Reply:
oops. got it. thanks.